korangar/src/loaders/model/mod.rs

```diff
@@ -389,12 +401,17 @@ impl ModelLoader {
         let mut root_nodes = vec![root_node];
 
         for root_node in root_nodes.iter_mut() {
-            Self::calculate_transformation_matrix(root_node, true, bounding_box, Matrix4::identity());
+            Self::calculate_transformation_and_static_matrix(root_node, true, bounding_box, Matrix4::identity());
         }
 
+        let is_static = root_nodes
+            .iter_mut()
+            .fold(true, |is_static, root_node| is_static & Self::is_static(root_node));
```

Comment:
If `is_static` doesn't mutate the state, it could be `any` instead of `fold`.

Reply:
There is an even better one called `all`.